packages/story-editor/src/components/canvas/karma/elementKeyboardNavigation.js

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { Fixture } from '../../../karma';
+import { useStory } from '../../../app/story';
+
+fdescribe('Canvas Element - keyboard navigation', () => {
+  let fixture;
+
+  beforeEach(async () => {
+    fixture = new Fixture();
+    fixture.setFlags({ floatingMenu: true });
+
+    await fixture.render();
+    await fixture.collapseHelpCenter();
+  });
+
+  afterEach(() => {
+    fixture.restore();
+  });
+
+  function getCanvasElementWrapperById(id) {
+    return fixture.querySelector(
+      `[data-testid="frameElement"][data-element-id="${id}"]`
+    );
+  }
+
+  it('should be able to navigate in and out of the floating menu with ctrl+alt+p & esc', async () => {
+    // Insert an element
+    const shapeTab = fixture.editor.library.shapesTab;
+    await fixture.events.click(shapeTab);
+    const rectangleShape = fixture.editor.library.shapes.shape('Rectangle');
+    await fixture.events.click(rectangleShape);
+
+    // Click on one of the added shapes
+    const elementId = await fixture.renderHook(() =>
+      useStory(({ state }) => state.currentPage.elements[1].id)
+    );
+    const wrapperEl = getCanvasElementWrapperById(elementId);
+    await fixture.events.click(wrapperEl);
+    expect(document.activeElement).toBe(wrapperEl);
+
```

Comment:
🤔 I think the shape element will be selected when you click on the button on line 49. This part of the test may not be necessary

Reply:
yup. was able to remove it. good catch